htdocs/core/lib/functions.lib.php

```diff
@@ -272,7 +272,7 @@ function GETPOST($paramname,$check='',$method=0,$filter=NULL,$options=NULL)
 	    }
 	}
 
-	return $out;
+	return htmlspecialchars($out);
```

Comment:
Since we still support PHP 5.3, I think that the character encoding should be specified to avoid character mangling.
Reference: https://php.net/manual/fr/function.htmlspecialchars.php#refsect1-function.htmlspecialchars-parameters